kolibri/core/templates/kolibri/base.html

```diff
@@ -7,6 +7,7 @@
   {% base_frontend_async %}
   {% base_frontend_sync %}
   <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
+  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
```

Comment:
Good to know! I wasn't sure how to do it since there were no examples and it's not mentioned in the kolibri docs, so I went with the easy staticfiles route.